.NET/Microsoft.Recognizers.Text.Number/Extractors/BasePercentageExtractor.cs

```diff
@@ -1,295 +1,301 @@
-﻿using System;

```

Comment:
This file seems fully updated due to changes in line breaks

Reply:
Feedback applied.